packages/teams/src/remove-team.ts

```diff
@@ -0,0 +1,23 @@
+import { UserSession } from "@esri/arcgis-rest-auth";
+import { removeGroup, unprotectGroup } from "@esri/arcgis-rest-portal";
+
+/**
+ * removeTeam unprotects a group, then deletes it
+ *
+ * @export
+ * @param {string} id Team Id
+ * @param {UserSession} authentication authentication
+ * @return {*}  {Promise<any>}
+ */
+export async function removeTeam(
```

Comment:
This method should be called deleteTeam

Remove ~ unassociate (but not delete)

Delete ~ destroy from system

Reply:
Let's handle the `remove` vs `destroy` vs `delete` in a wholistic manner -  for now let's leave this as `remove` since it's consistent w/ rest-js and the rest of hub.js